package.json

```diff
@@ -56,7 +56,7 @@
   "devDependencies": {
     "@types/babel-code-frame": "^6.20.0",
     "@types/chai": "^3.5.0",
-    "@types/commander": "^2.9.0",
+    "@types/commander": "^2.11.0",
```

Comment:
Do we even need this now that the package contains declaration files?

Reply:
Removed